mix.exs

```diff
@@ -36,7 +35,7 @@ defmodule Auth0Jwks.MixProject do
       {:joken, "~> 2.0"},
       {:joken_jwks, "~> 1.1.0"},
       {:plug_cowboy, "~> 2.0"},
-      {:poison, "~> 4.0"},
+      {:poison, "~> 4.0", optional: true}
```

Comment:
this is the last one. everything else was just `mix format`. hopefully this makes it easy to read.